pika/channel.py

```diff
@@ -332,11 +336,11 @@ def _generate_consumer_tag(self):
 
         :returns: consumer tag
         :rtype: str
+
         """
-        return 'ctag%i.%s' % (self.channel_number,
-                              uuid.uuid4().hex)
+        return 'ctag%i.%s' % (self.channel_number, uuid.uuid4().hex)
 
-    def basic_get(self, callback=None, queue='', no_ack=False):
+    def basic_get(self, queue='', no_ack=False, callback=None):
```

Comment:
`callback=None` in the function signature suggests that `callback` is optional. However, the semantics dictate that it's required as it doesn't make sense to call without one.. This is likely confusing to the API user.